modules/nested-accordion/widgets/nested-accordion.php

```diff
@@ -439,6 +439,23 @@ private function add_content_style_section() {
 			[
 				'name' => 'content_border',
 				'selector' => $low_specificity_accordion_item_selector,
+				'fields_options' => [
+					'color' => [
```

Comment:
We usually set the style defaults in the CSS file itself. 
But I guess this is valid as well.